Composer/packages/lib/shared/src/settings.ts

```diff
@@ -70,5 +70,8 @@ export const applyPublishingProfileToSettings = (settings: DialogSetting, profil
   settings.MicrosoftAppId = profile.settings.MicrosoftAppId;
   settings.MicrosoftAppPassword = profile.settings.MicrosoftAppPassword;
 
+  // apply the SCM domain host
```

Comment:
do we need this comment? looks straightforward here

Reply:
I was following the comments in the existing function code, but good point.  I don't like obvious comments either:)